JH5/task10/src/homework/Main.java

```diff
@@ -0,0 +1,82 @@
+package homework;
+
+import java.io.*;
+import java.util.HashMap;
+
+/**
+ * Class main has entry point to the program. Program create html-table,
+ * that shows time (ms) of request ip addresses.
+ */
+
+public class Main {
+
+    /**
+     * Address of working directory
+     */
+    public static final String ADDRESS = "C:\\Users\\777\\IdeaProjects\\task10\\src\\homework\\";
```

Comment:
variable name is wrong - it's not address it's look like dirpath or something like that